fiware-region-sanity-tests/sanity_checks

```diff
@@ -141,7 +141,12 @@ done
 }
 TESTS=${TESTS:-tests/regions}
 
-# Main
+# Delete all generated log files before each sanity check execution.
+echo "Deleting log files from previous executions"
+rm -f *.log
```

Comment:
I am thinking about this. We have a problem if Sanity Checks are parallelized. We must delete only the logs of the given regions by params (where Sanity Checks will be executed).